app/templates/stack/-certificate.hbs

```diff
@@ -2,12 +2,13 @@
   <div class="panel-heading with-actions">
     <h3>{{certificate.commonName}}</h3>
     <div class="panel-heading-actions">
+      {{#if certificate.isAcme}}
+        <span class="label label-success">Managed SSL</span>
+      {{/if}}
       {{#if certificate.inUse}}
-        {{#bs-tooltip placement="bottom" title="This certificate is currently in use and cannot be deleted." bs-container=false}}
-          <div>
-            <button class="btn btn-default btn-xs" disabled>Delete</button>
-          </div>
-        {{/bs-tooltip}}
+        {{! REVIEW: This used to have a tooltip, but it doesn't work on a
+            disabled object, and breaks layout. Are we OK losing it?}}
+        <button class="btn btn-default btn-xs" disabled>Delete</button>
```

Comment:
As a workaround, could we add an `<i class="fa fa-question-circle"></i>` next to the button and apply the tooltip there? I could see this being confusing for some users.  We could also do that in a separate PR as I know there are several cases of disabled buttons with broken tooltips.